integration-tests/tests.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+)
+
+func showQueryResult(output []entities.Data) {
+	fmt.Printf("\n\n")
+	fmt.Println("THING_ID SENSOR_ID VALUE TIMESTAMP")
+	for i := range output {
+		fmt.Println(output[i])
+	}
+}
+
+func startTest(msg error, testFunction error) {
+	status := "PASS"
+	err := testFunction
+	if err != nil {
+		status = "FAIL"
+	}
+	log.Printf("%s: %s", status, msg)
+
+}
+
+func validateOutputQuery(output []entities.Data, payload *thing) error {
```

Comment:
What is the purpose of this test? I think it's better to name the functions clearly, e.g. `SaveDataIntegrationTest`.

Also, it would be nice for each integration test bein able to run multiple test cases. This is quite similar to what we do in unit testing, as you can see [here](validateOutputQuery).

Each test case should have an clear description following this format:`When <something> happens <something> should <fail/succeed>`. In our context: `When an invalid data is published the storage should fail to save it`.

Reply:
Also, each test function must setup the needed data, publish whatever message it needs, and validate the result. Something similar to the AAA (arrange, act, assert) of unit testing.

Reply:
When you have to setup a lot of dependencies to run the tests, for example creating a connection to the database and message broker, this code could be moved to a `setupTests()` operation and then be reused across the integration tests.

Reply:
Done.